GETTING HELP.md

```diff
@@ -0,0 +1,14 @@
+Getting help with Etcher
+========================
+
+There are various ways you can get support for Etcher if you're having any issue, concern or idea you'd like to share with us.
```

Comment:
Here are some subjective style comments, I will appear rude in just writing them out directly from here on in but this one gets a preamble :P

>get support for Etcher if you're having any issue, concern or idea you'd like to share with us.

get support for Etcher if you're experience an issue or have an idea you'd like to share with us.